apps/api/src/routers/user.py

```diff
@@ -79,7 +79,7 @@ async def me(
 @router.post("/apply", status_code=status.HTTP_201_CREATED)
 async def apply(
     user: Annotated[User, Depends(require_user_identity)],
-    raw_application_data: Annotated[RawApplicationData, Depends(RawApplicationData)],
+    raw_application_data: Annotated[RawApplicationData, Form()],
     resume: Optional[UploadFile] = None,
```

Comment:
Seems this doesn't actually quite work, think the file upload is causing a conflict. Will look into this.